frontend/src/pages/operation_show/evidence_modals/index.tsx

```diff
@@ -145,18 +153,28 @@ export const EditEvidenceModal = (props: {
   const tagsField = useFormField<Array<Tag>>(props.evidence.tags)
   const codeblockField = useFormField<CodeBlock>({ type: 'codeblock', language: '', code: '', source: null })
   const adjustedAtField = useFormField<Date | undefined>(props.evidence.adjustedAt ?? undefined)
+  const c2EventField = useFormField<C2Event>({type: 'c2-event', c2: '', c2Operator: '', beacon: '', externalIP: '', internalIP: '', hostname: '', userContext: '', integrity: '', processName: '', processID: '', command: '', result: ''})
   React.useEffect(() => {
-    if (props.evidence.contentType !== 'codeblock') {
+      if (props.evidence.contentType !== 'codeblock' && props.evidence.contentType !== 'c2-event') {
```

Comment:
Would you mind running this file through a formatter? It looks like there is an extra indent here and some formatting changes below